crypto/cpu.c

```diff
@@ -0,0 +1,19 @@
+#include <openssl/cpu.h>
+
+#ifdef __linux__
```

Comment:
Yes, that seems right to me. However this is also ARM-specific code. I'd prefer it to be in the cpu-arm-linux.c file that you deleted, to keep that clear.